### PR TITLE
Fix np.concatenate not working properly

### PR DESCRIPTION
**Pull request by pavlin-policar**
I believe this is a bug in your code. I've adapted it to some data I'm currently working with and some of the ECG signals crashed this code. Changing this line to concatenate properly appears to fix the problem. I haven't looked at the algorithm at all, but suspect this is what is meant to happen anyway.

### CODE REVIEW

**Comment by nsunami** [comment]
Thank you for pointing this bug out! I ran your code, but I get a `ValueError ` 

> line 135, in <module>
>     ekg_window = np.concatenate([lows, i, highs])
>   File "<__array_function__ internals>", line 6, in concatenate
> ValueError: all the input arrays must have same number of dimensions, but the array at index 0 has 1 dimension(s) and the array at index 1 has 0 dimension(s)

I do agree that there is a bug in line 134 that the arrays are not concatenating properly.
Do you mean to instead change it to the following?

> ekg_window = np.concatenate((lows, i, highs), axis=None)

File: shannon_energy_rpd.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def running_mean(x, N):
     highs = np.arange(i+1, i+search_window_half+1)
     if highs[-1] > len(EKG):
         highs = np.delete(highs, np.arange(np.where(highs == len(EKG))[0], len(highs)+1))
-    ekg_window = np.concatenate(lows, i, highs)
+    ekg_window = np.concatenate([lows, i, highs])
```
Comment (nsunami) [inline shannon_energy_rpd.py:134]: I think, was meant to be:
`ekg_window = np.concatenate((lows, i, highs), axis=None)`

Comment (pavlin-policar) [reply shannon_energy_rpd.py:134]: My bad, I haven't actually been running the script in this repo, I've just been copying out the algorithm and running it on my data.

Actually it was meant to be this:
```
ekg_window = np.concatenate((lows, [i], highs))
```

Comment (nsunami) [reply shannon_energy_rpd.py:134]: Got it thanks! Can you update the commit with the new code so that I can merge the request?

**Review by nsunami — CHANGES_REQUESTED** [review]
Thanks for working on this! I will mark this as "request changes" before it gets merged

**Review by nsunami — APPROVED** [review]

**Pull request merged**